src/Integration/Laravel/Filesystem/composer.json

```diff
@@ -15,8 +15,8 @@
         "async-aws/core": "^1.0",
         "async-aws/flysystem-s3": "^1.0",
         "async-aws/simple-s3": "^1.0",
-        "illuminate/filesystem": "^6.18.13 || ^7.10 || ^8.0",
-        "illuminate/support": "^6.18.13 || ^7.10 || ^8.0"
+        "illuminate/filesystem": "^6.18.13 || ^7.10 || ^8.0 || ^9.0 || ^10.0",
```

Comment:
this doesn't work. the laravel 9 and 10 filesystem components are massively different.